templates/buy_together_app/help_parts/nav.html

```diff
@@ -1,11 +1,36 @@
-<nav class="navbar navbar-expand-lg navbar-dark bg-dark mb-5">
+<!DOCTYPE html>
+<html>
+    
+<head>
+	<title>Login</title>
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP" crossorigin="anonymous">
+	<link rel="stylesheet" href="/static/css/nav_bar.css">
+</head>
+
+
+<body>
+  <nav class="navbar navbar-expand-lg navbar-dark bg-dark mb-5">
     <div class="container">
             <a class="text-primary" href="/"><h1>Home</h1></a>
-            {% if request.user.is_authenticated %}
-              <p style="color: beige;">hello {{ request.user.username }}!</p>
-            {% else %}
-              <a class="text-primary" href="/signup"><h5>Sign Up</h5></a>
-            {% endif %}
+            <div class="container nav_sides">
+              {% if request.user.is_authenticated %}
+                <p class="nav_side_element">hello {{ request.user.username }}!</p>
+
+                {% if supplier %}
+                <form><a class="btn btn-primary nav_side_element" href="/suppliers" onclick="location.href='/my-page'">My Page</a></form>
+                {% endif %}
+
+                {% if client %}
```

Comment:
In the context there is no client,if you log in as client i guess you will get only the request.user.name with out the tag <a>

Reply:
ive just created a set up for the client, once the client page will be merged i will update it.